app-arch/bzip2/bzip2-1.0.6.0.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic ltprune multilib-minimal
+
+DESCRIPTION="A high-quality data compressor used extensively by Gentoo Linux"
+HOMEPAGE="http://www.bzip.org/"
+SRC_URI="http://ftp.suse.com/pub/people/sbrabec/bzip2/tarballs/${P}.tar.gz"
+
+LICENSE="BZIP2"
+SLOT="0/1" # subslot = SONAME
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~sparc-fbsd ~x86-fbsd"
+IUSE="static static-libs"
+
+DOCS=( manual.pdf NEWS README.autotools README README.XML.STUFF )
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.0.6-progress.patch
+	"${FILESDIR}"/${PN}-1.0.4-POSIX-shell.patch #193365
+	"${FILESDIR}"/${PN}-1.0.6-mingw.patch #393573
+	"${FILESDIR}"/${PN}-1.0.6-CVE-2016-3189.patch #620466
+)
+
+multilib_src_configure() {
+	use static && append-ldflags -static # doesn't seem to work...
+
+	ECONF_SOURCE="${S}" econf \
+		$(use_enable static-libs static)
+}
+
+multilib_src_install() {
+	emake DESTDIR="${ED}" install
+}
+
+multilib_src_install_all() {
+	prune_libtool_files
+	default
+	mkdir -p ${ED}/bin
+	for i in bzip2$(get_exeext) bunzip2$(get_exeext) bzcat$(get_exeext); do
+		mv ${ED}/usr/bin/$i ${ED}/bin/ || die
+	done
+}
```

Comment:
What has happen to all the *dosym \** stuff?

Reply:
For the binaries, dosym is not needed. I'm looking into the man pages.